data/expression/script/gocc/fs.bnf

```diff
@@ -23,8 +22,9 @@ _escapeChar : '\\' 'n' | '\\' 'r' | '\\' 't' ;
 
 _dq_string  : '"' { . | '\\' '"' | _escapeChar } '"' ;
 _sq_string  : '\'' { . | '\\' '\'' | _escapeChar } '\'' ;
+_lit_string : '`' { . } '`' ;
 
-stringLit   :  _dq_string | _sq_string ;
+stringLit   :  _dq_string | _sq_string | _lit_string ;
 
```

Comment:
other than support for _lit_string, are the rest of the bnf changes required given that were moving to a json path as a function?

Reply:
The other bnf changes add support for recursive dereference: $.foo.bar.x.y[3].z; the other changes would also allow an AST of a Ref to be built so that Refs are not parsed twice.